docs/configuration.md

```diff
@@ -328,18 +640,88 @@ reporting:
 		name: newrelicname
 ```
 
-The reporting option is **optional** and configures error and metrics reporting tools. At the moment only two services are supported, New Relic and Bugsnag, a valid configuration may contain both.
+The `reporting` option is **optional** and configures error and metrics
+reporting tools. At the moment only two services are supported, [New
+Relic](http://newrelic.com/) and [Bugsnag](http://bugsnag.com), a valid
+configuration may contain both.
 
 ### bugsnag
 
-- apikey: **Required** - API Key provided by Bugsnag
-- releasestage: **Optional** - TODO: fill in description
-- endpoint: **Optional** - TODO: fill in description
+<table>
+  <tr>
+    <th>Parameter</th>
+    <th>Required</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>
+      <code>apikey</code>
+    </td>
+    <td>
+      yes
+    </td>
+    <td>
+      API Key provided by Bugsnag
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <code>releasestage</code>
+    </td>
+    <td>
+      no
+    </td>
+    <td>
+      Tracks where the registry is deployed, for example,
+      <codde>production</code>,<codde>staging</code>, or
+      <codde>development</code>.
+    </td>
+  </tr>  
+  <tr>
+    <td>
+      <code>endpoint</code>
+    </td>
+    <td>
+      no
+    </td>
+    <td>
+      Specify the enterprise Bugsnag endpoint. 
+    </td>
+  </tr>  
+</table>
+
 
 ### newrelic
 
-- licensekey: **Required** - License key provided by New Relic
-- name: **Optional** - New Relic application name
+<table>
+  <tr>
+    <th>Parameter</th>
+    <th>Required</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>
+      <code>licensekey</code>
+    </td>
+    <td>
+      yes
+    </td>
+    <td>
+      License key provided by New Relic.
+    </td>
+  </tr>
+   <tr>
+    <td>
+      <code>name</code>
+    </td>
+    <td>
+      no
+    </td>
+    <td>
+      New Relic application name.
+    </td>
+  </tr> 
+</table>
```

Comment:
There is a third option available to newrelic:
verbose bool : enable newrelic debugging output on stdout

see https://github.com/docker/distribution/commit/c359a6d1726c2547ceb5914a3a3fb3c5acaea46f